addons/web/static/tests/views/list/list_view.test.js

```diff
@@ -3426,9 +3426,9 @@ test(`selection box is properly displayed (multi pages)`, async () => {
     await mountView({
         resModel: "foo",
         type: "list",
-        arch: `<list limit="3"><field name="foo"/><field name="bar"/></list>`,
+        arch: `<list limit="2" count_limit="3"><field name="foo"/><field name="bar"/></list>`,
```

Comment:
Do not change this test either. In your case, the single/multi page cases are similar. It's ok to test only one of them. You can introduce a new test for that.